contrib/IECoreUSD/src/IECoreUSD/bindings/IEUSDModule.cpp

```diff
@@ -32,8 +32,24 @@
 //
 //////////////////////////////////////////////////////////////////////////
 
+#include "IECoreUSD/DataAlgo.h"
+
 #include "boost/python.hpp"
 
+using namespace boost::python;
+using namespace IECoreUSD;
+
+namespace IECoreUSDBindings
```

Comment:
I think this namespace is redundant, since it only contains a `BOOST_PYTHON_MODULE`, and that is declared `extern "C"`.

Reply:
Addressed in 8e4c777fc41b150dcee66d5775e4261470f666e1